tests/lib/prepare-restore.sh

```diff
@@ -381,6 +381,8 @@ prepare_project() {
             if [ -d /var/lib/snapd ]; then
                 echo "# /var/lib/snapd"
                 ls -lR /var/lib/snapd || true
+                journalctl -b | tail -100 || true
+                cat /var/lib/snapd/state.json || true
```

Comment:
Maybe | json_pp it? I fear it will be hard to understand otherwise,

Reply:
Yeah, we discussed this briefly on irc. I think it's ok to manually copy to jq for now but it might be something to do eventually. But hopefully we find the root cause of this and then we can get rid of all this silly debug :)